wazo_calld/plugins/voicemails/api.yml

```diff
@@ -376,10 +358,7 @@ paths:
           $ref: '#/responses/AnotherServiceUnavailable'
     head:
       summary: Check if a custom greeting exist
-      description: '**Required ACL:** `calld.users.me.voicemails.greetings.{greeting}.read`
-
-
-        **Not implemented**'
+      description: '**Required ACL:** `calld.users.me.voicemails.greetings.{greeting}.read`'
```

Comment:
The HEAD doesn't seem implemented?

Reply:
I wonder if that make sense to do it, as asterisk doesn't provide HEAD neither.

@fblackburn1 @pc-m @nballas-wazo What do you think, we implement it in Asterisk or we drop it here ?

Reply:
for record, solution is to drop it until we need it